### PR TITLE
[ESS] [8.16] Ingest Sysdig Falco data to Elasticsearch

### LINKED ISSUE #6004: [CloudSecurity][Docs] Streamlined CNCF Falco ingestion documentation
## Background

https://github.com/elastic/security-team/issues/9873
https://github.com/elastic/security-team/issues/9980
https://github.com/elastic/security-team/issues/10379

## Documentation Content

Detailed technical setup has been captured [here](https://docs.google.com/document/d/1ufaJTH4Ou-U28Np2Ul4J_EJCQ3iTJ9evTdCnpO68udU/edit?usp=sharing).

### Background & resources

* PRs: 
* Issues/metas: 
* Point of contact: @nick-alayil 
* Test environments: 

### Which documentation set does this change impact?

ESS only

### ESS release

8.16

### Serverless release

N/A

### Feature differences

N/A

### API docs impact

 N/A

### Prerequisites, privileges, feature flags

 N/A

### PR DESCRIPTION
**Pull request by benironside**
Fixes #6004 by creating a new guide for ingesting third-party alert data from cloud security tool Sysdig Falco, and a new section that contains that guide.

Previews: [Ingest Falco data](https://security-docs_bk_6027.docs-preview.app.elstc.co/guide/en/security/master/ingest-falco.html); [Ingest third-party cloud security data 
](https://security-docs_bk_6027.docs-preview.app.elstc.co/guide/en/security/master/ingest-third-party-cncf.html)

### CODE REVIEW

File: docs/cloud-native-security/falco-setup.asciidoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+[discrete]
+=== Configure Falcosidekick:
+
+. Refer to Falcosidekick's documentation to https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[install Falcosidekick].
+. Use the https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[Falcosidekick on Docker using environment variables] method and set your environment variables as follows:
+.. `ELASTICSEARCH_HOSTPORT`: your {es} endpoint URL, which can be found under **Connection details** on the upper right of the **Integrations** page in {kib}.
+.. `ELASTICSEARCH_INDEX`: the {es} index where you want to store Falco logs.
+.. `ELASTICSEARCH_SUFFIX`: The frequency with which you want the {es} index suffix to change. Either `daily`, `monthly`, `annually`, or `none`. 
+.. `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD`: the username and password for an account on your {es} instance, with at least the following permissions: Which permissions, Nick?
```
Comment (nick-alayil) [inline docs/cloud-native-security/falco-setup.asciidoc:54]: > Which permissions, Nick?

For both username/password and API key, we should note that user should follow the principles of least privilege, with permissions limited to creating and writing to Falco indices. Maybe we could point user to [this](https://www.elastic.co/guide/en/kibana/current/api-keys.html) for API key and for username/password I assume there is a page similar to APIkey, which I couldn't locate. The closet one I could locate is [this](https://www.elastic.co/guide/en/elasticsearch/reference/current/setting-up-authentication.html)

**Review by nick-alayil — COMMENTED** [review]
Overall this is an awesome start 🚀 , @benironside. Your efforts are much appreciated 🙏. .

File: docs/cloud-native-security/falco-setup.asciidoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+. Find the **Rules** page in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. Click **Detection rules (SIEM)**.
+. Search for a rule called `External Alerts`. Enable it using the **Enabled** toggle on the right.
+
+image::images/falco-external-alerts-rule.gif[A gif showing the steps to enable the External Alerts rule]
+
+[discrete]
+[[ingest-falco-setup-falco]]
+== Configure Falco and Falcosidekick
```
Comment (nick-alayil) [inline docs/cloud-native-security/falco-setup.asciidoc:33]: Perhaps we can split this into two sections: the instructions below are for VMs, while for k8s, it's much simpler using Helm chart commands, which you can find in the doc in the initial ticket. I’ve included them below for convenience.
```
# Add falco helm charts
helm repo add falcosecurity https://falcosecurity.github.io/charts
helm repo update

# Install falco and falcosidekick with one chart
helm install falco falcosecurity/falco \
        --set falcosidekick.enabled=true \
        --set tty=true \
        --set driver.kind=modern_ebpf \
        --set collectors.kubernetes.enabled=true \
        --set falcosidekick.config.elasticsearch.hostport="https://<ES host>" \
        --set falcosidekick.config.elasticsearch.username="elastic" \
    --set falcosidekick.config.elasticsearch.password="<password>" \
        --set falcosidekick.config.elasticsearch.index="logs-falco.alerts-all" \
        --set falcosidekick.config.elasticsearch.suffix="none"
```

**Comment by benironside** [comment]
continuing this work in https://github.com/elastic/security-docs/pull/6046 since it touched related pages

**Comment by benironside** [comment]
closing this issue since the work was added to #6046

**Pull request closed without merging**